### ISSUE TITLE
Make a helper script for theme renaming

### ISSUE THREAD

**Issue by ltade** [Automattic/_s#1564]
> 
>  1.    Search for '_s' (inside single quotations) to capture the text domain and replace with: 'megatherium-is-awesome'.
>  1.   Search for _s_ to capture all the functions names and replace with: megatherium_is_awesome_.
>  1.   Search for Text Domain: _s in style.css and replace with: Text Domain: megatherium-is-awesome.
>  1.   Search for  _s (with a space before it) to capture DocBlocks and replace with:  Megatherium_is_Awesome.
>  1.   Search for _s- to capture prefixed handles and replace with: megatherium-is-awesome-.
>  1.  Search for _S_ (in uppercase) to capture constants and replace with: MEGATHERIUM_IS_AWESOME_.
> 

Its good there there is instructions for this, but wouldn't it be more convenient of we had a script that did that?

**Comment by lancewillett** [comment]
Yes, there is a handy script available at https://underscores.me/
